easybuild/easyconfigs/s/Spack/Spack-0.12.1.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'Spack'
+version = '0.12.1'
+
+homepage = 'https://spack.io/'
+description = """Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific
+ software easy. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,
+ and all of these builds can coexist on the same machine."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/LLNL/spack/archive/']
```

```suggestion
source_urls = ['https://github.com/spack/spack/archive/']
```
It should redirect, but this was updated a while ago so might as well be consistent.